src/detail-page/DetailPage.tsx

```diff
@@ -26,7 +25,15 @@ export const DetailPage: React.FC<{}> = () => {
     <>
       {profession && (
         <MainContainer>
```

Comment:
We need some space between the wrapper and card, now card displays on the top of the website, so we could add padding-top to the wrapper.

Reply:
I added top padding to the content container.